functions/backup.bash

```diff
@@ -48,9 +48,8 @@ restore_openhab_config() {
   local backupPath="${OPENHAB_BACKUPS:-/var/lib/openhab/backups}"
   local filePath
   local fileSelect
-  local introText="This will restore a backup of your openHAB configuration using openHAB's builtin backup tool.\\n\\nWould you like to continue?"
```

Comment:
Why remove the intro text and first confirmation question, I think it is a good thing to have as if a user is not careful they could overwrite their config unintentionally.

Reply:
They need to select the file to restore from anyway (can cancel there of course, too) so it was a quite unnecessary extra input.

Reply:
OK, I'm ok with it.